README.md

```diff
@@ -1,3 +1,125 @@
 [![Build Status](https://travis-ci.org/TDycores-Project/TDycore.svg?branch=master)](https://travis-ci.org/TDycores-Project/TDycore)
 [![Code Coverage](https://codecov.io/gh/TDycores-Project/TDycore/branch/master/graph/badge.svg)](https://codecov.io/gh/TDycores-Project/TDycore)
 
+# The Terrestrial Dynamical Core (TDycore)
+
+This project consists of a set of dynamical cores for simulating
+three-dimensional land processes in order to address questions about the
+hydrological cycle at global scale. TDycore applies higher-order,
+spatially-adaptive algorithms on geometries that capture features of interest,
+in an attempt to study problems including
```

Comment:
Due to rescoping, we are only focusing on subsurface flow. Thus, I would restate the application of TDycore as the text below and remove the itemized list:

```
TDycore applies higher-order,
spatially-adaptive algorithms on geometries that capture features of interest
to study subsurface transport of water and energy.
```